### PR TITLE
Add expanded capacity display table

### PR DESCRIPTION
**Pull request by Camwyn**
Ticket: https://central.tri.be/issues/80619

Adds a table displaying capacity to settings panel.

Requires: moderntribe/event-tickets-plus#342 (ET+)

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$header_img' | [meta-box.php:260](https://github.com/moderntribe/event-tickets/blob/8a3fdb122594e598f32b605e49f584a3d61f0362/src/admin-views/meta-box.php#L260)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$header_img' | [meta-box.php:244](https://github.com/moderntribe/event-tickets/blob/3dccf9c5f8aa4d9454f5a37ac0d90f37c7f7a7d5/src/admin-views/meta-box.php#L244)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$header_img' | [meta-box.php:244](https://github.com/moderntribe/event-tickets/blob/0e1b1bc7d7b1e237d6090c23e9e05223237071b6/src/admin-views/meta-box.php#L244)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Tickets_Handler.php [hunk 1]
```diff
@@ -169,6 +169,158 @@ public function get_total_event_capacity( $post = null ) {
...
+	public function get_total_event_independent_capacity( $post = null ) {
+		$post_id = Tribe__Main::post_id_helper( $post );
+
+		$capacity = 0;
+
+		$tickets = Tribe__Tickets__Tickets::get_event_tickets( $post_id );
+
+		if ( ! empty( $tickets ) ) {
+			foreach ( $tickets as $ticket ) {
+				if ( 'own' != $ticket->global_stock_mode() ) {
```
Comment (bordoni) [inline src/Tribe/Tickets_Handler.php]: Use `!==`

File: src/Tribe/Tickets_Handler.php [hunk 2]
```diff
@@ -169,6 +169,158 @@ public function get_total_event_capacity( $post = null ) {
...
+		}
+
+		/**
+		 * Allow templates to filter the returned value
+		 *
+		 * @param (int) $capacity Total capacity value
+		 * @param (int) $post Post ID tickets are attached to
+		 * @param (array) $tickets array of all tickets
+		 *
+		 * @since TDB
```
Comment (bordoni) [inline src/Tribe/Tickets_Handler.php]: Move `@since` above `@param`

File: src/Tribe/Tickets_Handler.php [hunk 3]
```diff
@@ -169,6 +169,158 @@ public function get_total_event_capacity( $post = null ) {
...
+		 */
+		return apply_filters( 'tribe_tickets_total_event_independent_capacity', $capacity, $post_id, $tickets );
+	}
+
+	/**
+	 * Get an array list of independent ticket names for an event.
+	 *
+	 * @param $event_id int (null)
+	 *
+	 * @since TBD
```
Comment (bordoni) [inline src/Tribe/Tickets_Handler.php]: Move `@since` above `@param`

File: src/Tribe/Tickets_Handler.php [hunk 4]
```diff
@@ -169,6 +169,158 @@ public function get_total_event_capacity( $post = null ) {
...
+
+		return $ticket_list;
+	}
+
+	/**
+	 * Get the total event shared capacity.
+	 *
+	 * @param $event_id int (null)
+	 *
+	 * @since TBD
```
Comment (bordoni) [inline src/Tribe/Tickets_Handler.php]: Move `@since` above `@param`

File: src/Tribe/Tickets_Handler.php [hunk 5]
```diff
@@ -169,6 +169,158 @@ public function get_total_event_capacity( $post = null ) {
...
+		 */
+		return apply_filters( 'tribe_tickets_total_event_shared_capacity', $capacity, $post_id, $tickets );
+	}
+
+	/**
+	 * Get an array list of shared ticket names for an event.
+	 *
+	 * @param $event_id int (null)
+	 *
+	 * @since TBD
```
Comment (bordoni) [inline src/Tribe/Tickets_Handler.php]: Move `@since` above `@param`

File: src/Tribe/Tickets_Handler.php [hunk 6]
```diff
@@ -169,6 +169,158 @@ public function get_total_event_capacity( $post = null ) {
...
+		}
+
+		/**
+		 * Allow templates to filter the returned value
+		 *
+		 * @param (int) $capacity Total capacity value
+		 * @param (int) $post Post ID tickets are attached to
+		 * @param (array) $tickets array of all tickets
+		 *
+		 * @since TDB
```
Comment (bordoni) [inline src/Tribe/Tickets_Handler.php]: Move `@since` above `@param`

File: src/admin-views/meta-box.php [hunk 7]
```diff
@@ -30,27 +32,18 @@
...
-					</span>
-				</span>
+
 				<?php
 				/**
-				 * Allows for the insertion of additional elements into the main ticket admin panel "header"
+				 * Allows for the insertion of total capacity element into the main ticket admin panel "header"
+				 *
+				 * @param Post ID
+				 * @since TBD
```
Comment (bordoni) [inline src/admin-views/meta-box.php]: Move `@since` above `@param`

File: src/admin-views/meta-box.php [hunk 8]
```diff
@@ -30,27 +32,18 @@
...
+				 */
+				do_action( 'tribe_events_tickets_capacity', $post_id );
+
+				/**
+				 * Allows for the insertion of additional elements (buttons/links) into the main ticket admin panel "header"
 				 *
 				 * @param int $post_id the id of the post
 				 * @since TBD
```
Comment (bordoni) [inline src/admin-views/meta-box.php]: Move `@since` above `@param`

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$header_img' | [meta-box.php:244](https://github.com/moderntribe/event-tickets/blob/0f96ac9116defddb16993229c981b2e53bdcdabb/src/admin-views/meta-box.php#L244)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$header_img' | [meta-box.php:246](https://github.com/moderntribe/event-tickets/blob/14d9db4ea36e619fc426e5f12c2bbbdcf3350568/src/admin-views/meta-box.php#L246)

> via jenkins/codesniffer-phpcs

**Comment by Camwyn** [comment]
Merged in bucket, needs another review 👍  please!

**Pull request merged**